admin/server/middleware/browserify.js

```diff
@@ -5,6 +5,7 @@ var moment = require('moment');
 var packages = require('../../client/packages');
 var path = require('path');
 
+var babelrc = fs.readJsonSync(path.resolve(__dirname, '../../../.babelrc'), { throws: false }) || {};
```

Comment:
Unless I'm mistaken, this is so that custom fields pick up the babelrc from the keystone root, correct?

Reply:
correct